documentation/src/main/resources/theme/default-theme.yml

```diff
@@ -0,0 +1,271 @@
+font:
```

Comment:
Can you add a reference to the original source of this (GitHub link). And instead having the two files side by side, how about adding the original one verbatim in one commit and then adapt it as per our needs in a separate commit. That's how we handled other cases where started from a copied resource.